inference-engine/samples/speech_sample/main.cpp

```diff
@@ -857,7 +857,10 @@ int main(int argc, char *argv[]) {
 
                 uint32_t numFramesReference(0), numFrameElementsReference(0), numBytesPerElementReference(0),
                         numBytesReferenceScoreThisUtterance(0);
-                const uint32_t numScoresPerFrame = ptrOutputBlob.size() / batchSize;
+                auto dims = outputs.empty() ? cOutputInfo.rbegin()->second->getDims() : cOutputInfo[outputs[next_output]]->getDims();
+                const uint32_t numScoresPerFrame = std::accumulate(std::begin(dims), std::end(dims), (size_t)1, std::multiplies<size_t>());
```

Comment:
please use static_cast for (size_t) instead of C-style cast or even better size_t{1}.